packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -146,6 +152,66 @@ export default function TranslationView() {
 
   const userCan = useAccessControl();
 
+  const firstWord = useRef<HTMLInputElement>(null);
+  const lastWord = useRef<HTMLInputElement>(null);
+  const handleKeyPress = useCallback(
+    (event: {
```

Comment:
this can be typed as `KeyboardEvent`

Reply:
That causes errors, because I use this callback from two different places, which use two different types called `KeyboardEvent`. The one place calls it with a `KeyboardEvent`, the other place calls it with `KeyboardEvent<HTMLElement>`. This type I supplied here contains just the properties that I'm interested in, which the two types share.

The react `KeyboardEvent` seems to conflict with the builtin [`KeyboardEvent`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent)
<img width="690" alt="image" src="https://github.com/arrocke/gloss-translation/assets/17209828/e9aec048-f5cb-4f27-990d-0700d6e4427d">

Reply:
Ah, that makes more sense